src/prefect/core/flow.py

```diff
@@ -805,6 +806,21 @@ def edges_from(self, task: Task) -> Set[Edge]:
             )
         return self.all_downstream_edges()[task]
 
+    def is_mapped(self, task: Task) -> bool:
```

Comment:
This may suggest that it's checking whether the flow is mapped - to be more explicit, can this be `is_mapped_task`?

Reply:
sure, I was trying to be consistent with `flow.edges_from(task)`, which allows it to be read like a sentence.